pkg/cmd/resource/operation.go

```diff
@@ -69,8 +69,33 @@ func (oc *OperationCmd) runOperationCmd(cmd *cobra.Command, args []string) error
 
 	oc.Parameters.AppendData(flagParams)
 
-	_, err = oc.MakeRequest(apiKey, path, &oc.Parameters, false)
+	if oc.ConfirmAccount {
+		// display account information anf confirm whether user wants to proceed
```

Comment:
```suggestion
		// display account information and confirm whether user wants to proceed
```

Reply:
Thanks @richardm-stripe for spotting the mistake! I have committed the change